### PR TITLE
Big sync

### PR DESCRIPTION
**Pull request by doctorfree**
This is a very large pull request, over 100 commits. It's really too large to be a single pull request. I apologize.

I've merged cheat sheets from your repository with mine and a couple of other cheat sheet repositories. All are MIT licensed so the only restriction is retention of copyright notices.

The changes include:

- New cheat sheets
- Revised cheat sheets
    - Wiki style links are replaced with Markdown style links
    - Some expanded sections
    - Tags and links to other cheat sheets
    - CommonMark spec conformance
- Top level README and LICENSE
    - Added copyright notices for all authors
    - Combined README info

You may wish to selectively merge some of these changes and reject or modify others. Or, you may wish to ignore and close this pull request and ask me to submit others with more isolated changes easier to merge. I am ok with however you want to deal with it. I just wanted to get you my changes as my time will be shifted to another project for a while.

Anyway, it's cool what you have done and I wanted to contribute my work. I just am doing it in a clumsy manner due to time constraints. Feel free to pick and choose or toss. Thanks!

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -1,3 +1 @@
-# These are supported funding model platforms
-
-patreon: christianlempa
+github: doctorfree
```
Comment (thedatabaseme) [inline .github/FUNDING.yml:1]: I don't think that's what @ChristianLempa had in mind.

**Comment by doctorfree** [comment]
It looks like this pull request included my `.github/FUNDING.yml` which of course would be inappropriate for your repository.

Please ignore. My bad.

**Comment by ChristianLempa** [comment]
Wow, this is a lot. I need more time to review it.

**Comment by jrobiii** [comment]
Hi @doctorfree, I spoke with @ChristianLempa about reviewing PRs and Issues and he was good with it, but I won't have a large enough block of time in the next 6 months (at least) to review this one.  

Would you consider breaking this PR into smaller PRs?

**Comment by doctorfree** [comment]
@jrobiii yes, I was already considering doing that. This PR was too large and I noted that when I submitted it but I just wanted to get these changes under review or anyway submitted. I guess this PR is sort of a placeholder for me. We can close it and I will submit more easily digestible PRs.

On a related topic, a significant portion of the changes in this PR were the replacement of wiki-style links with CommonMark spec format links. I will continue to use CommonMark spec links in any submitted PRs but wiki-style links are scattered throughout and, in my opinion, should be replaced. Unfortunately, it seems the Obsidian community mostly uses wiki-style links and this repository is primarily designed as an Obsidian vault.

**Comment by jrobiii** [comment]
Thanks for the update!

I just noticed that @ChristianLempa just merged PR #92 pertaining to Issue #86, specifically about the GitHub link style versus the Obsidian markdown internal link style.  You may want to review those for some background.

**Pull request closed without merging**